testing/matrix/linter/rules/modules/prometheus_rules.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2022 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package modules
+
+import (
+	"crypto/sha256"
+	"os"
+	"os/exec"
+	"sync"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/deckhouse/deckhouse/testing/matrix/linter/rules/errors"
+	"github.com/deckhouse/deckhouse/testing/matrix/linter/storage"
+	"github.com/deckhouse/deckhouse/testing/matrix/linter/utils"
+)
+
+type checkResult struct {
+	success bool
+	errMsg  string
+}
+
+const promtoolPath = "/deckhouse/bin/promtool"
+
+var (
+	rulesCache   = map[string]checkResult{}
+	rulesCacheMu = sync.RWMutex{}
+)
+
+func promtoolAvailable() bool {
+	info, _ := os.Stat(promtoolPath)
+	return info != nil && (info.Mode().Perm()&0111 != 0)
+}
+
+func marshalChartYaml(object storage.StoreObject) (marshal []byte, hash string, err error) {
+	marshal, err = yaml.Marshal(object.Unstructured.Object["spec"])
+	if err != nil {
+		return
+	}
+	hash = string(newSHA256(marshal))
+	return
+}
```

Comment:
```suggestion
func marshalChartYaml(object storage.StoreObject) ([]byte, string, error) {
	marshal, err := yaml.Marshal(object.Unstructured.Object["spec"])
	if err != nil {
		return nil, "", err
	}
	return marshal, string(newSHA256(marshal)), nil
}
```

Reply:
This suggestion conflicts with previous one due to changed `newSHA256()` signature, so I cannot apply it directly.
I'll add corresponding change manually.
BTW, does named return parameters considered bad practice?

Reply:
I've addresed this request in 4b92715708a0d0b324c42edfc3952b24edb332a0.